velox/common/memory/ArbitrationParticipant.cpp

```diff
@@ -31,21 +31,24 @@ using namespace facebook::velox::memory;
 
 std::string ArbitrationParticipant::Config::toString() const {
   return fmt::format(
-      "minCapacity {}, fastExponentialGrowthCapacityLimit {}, slowCapacityGrowRatio {}, minFreeCapacity {}, minFreeCapacityRatio {}",
-      minCapacity,
-      fastExponentialGrowthCapacityLimit,
+      "initCapacity {}, minCapacity {}, fastExponentialGrowthCapacityLimit {}, slowCapacityGrowRatio {}, minFreeCapacity {}, minFreeCapacityRatio {}",
+      succinctBytes(initCapacity),
+      succinctBytes(minCapacity),
+      succinctBytes(fastExponentialGrowthCapacityLimit),
       slowCapacityGrowRatio,
-      minFreeCapacity,
+      succinctBytes(minFreeCapacity),
```

Comment:
Can we add memoryPool back to prefix? It will help with readability. Also could you please make sure to make followup changes to all the same meaning name across the codebase to make it consistent?